jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java

```diff
@@ -688,7 +688,7 @@ public Response payAllInvoices(@PathParam("accountId") final String accountId,
                                            invoice.getBalance() : remainingRequestPayment;
             if (amountToPay.compareTo(BigDecimal.ZERO) > 0) {
                 final UUID paymentMethodId = externalPayment ? null : account.getPaymentMethodId();
-                createPurchaseForInvoice(account, invoice.getId(), amountToPay, paymentMethodId, externalPayment, pluginProperties, callContext);
+                createPurchaseForInvoice(account, invoice.getId(), amountToPay, paymentMethodId, externalPayment, UUIDs.randomUUID().toString(), UUIDs.randomUUID().toString(), pluginProperties, callContext);
```

Comment:
@pierre 
I've made some changes. 
Please take a look and let me know your thoughts.

Thanks!